src/main/javacc/copy.jjt

```diff
@@ -255,7 +258,9 @@ void identifier() #ID :
 {
   t=<IDENTIFIER>
   {
-    jjtThis.setName(t.image.toLowerCase());
+    // Only support US_ASCII for now. UTF-8 support will be added later.
```

Comment:
two requests:
1. can you explicitly check for escaped unicode -- anything that starts with U& or u&
2. can you explicitly set the error message to something like "Unicode identifiers are not supported" instead of relying on the StandardCharsets library

Reply:
Input form STDIN is not escaped.  Any strings with U& or u& will not be interpreted as unicode but just those characters instead.  The user would have to manually put the unicode characters into the input in which case the lexer would not match it as an identifier. We only allow ASCII at the moment.

Reply:
Done.

Added tests for this as well.